store/cart.ts

```diff
@@ -0,0 +1,27 @@
+import { create } from 'zustand';
+
+import { SingleProductCardType } from '@/types/product.types';
+
+interface useCardType {
+  cardItems: SingleProductCardType[];
+  // eslint-disable-next-line no-unused-vars
+  addItemToCard: (item: SingleProductCardType) => void;
+}
+
+const useCard = create<useCardType>((set, get) => ({
+  cardItems: [],
+  addItemToCard: (item: SingleProductCardType) => {
+    console.log(get().cardItems);
+    const isExist = get().cardItems.find((cardItem) => cardItem.id === item.id);
+    if (isExist) {
+      if (typeof isExist.quantity === 'number') {
```

Comment:
Here we still need to rename card to cart.

Reply:
I already change them, look at the second open pull request